frontend/src/static/index.html

```diff
@@ -65,6 +65,9 @@
       gtag('config', '$GOOGLE_ANALYTICS_ID');
     </script>
     <script src="/public/js/index.js"></script>
+    <!-- To test with real data:
+          "$API_URL" -> "https://api.webstatus.dev"
+        -->
```

Comment:
```suggestion
```

Instead, we should tell the developer to change it [here](https://github.com/GoogleChrome/webstatus.dev/blob/main/frontend/manifests/pod.yaml#L33) in the https://github.com/GoogleChrome/webstatus.dev/blob/main/DEVELOPMENT.md document

This will keep all the local configuration in one place and reduce potential confusion of where configs are being set.

Reply:
Good point.  This will also encourage us to keep the documentation up-to-date.

Reply:
done